.github/workflows/ci.yml

```diff
@@ -37,13 +37,14 @@ jobs:
       - name: Validate testplans with schema
         run: ./ci/scripts/validate_testplans.sh
       - name: C/C++ formatting
-        run: ./ci/scripts/clang-format.sh "$GITHUB_BASE_REF"
+        run: ./ci/bazelisk.sh test //quality:clang_format_check
```

Comment:
This one takes ~25 seconds, so might worth consider optimizing in the future -- not blocking for this PR.

Reply:
I won't fix this for this PR, but in the future I hope to adopt `rules_lint` which will replace this and might be faster.

Reply:
I think this is not the same semantic? ``clang_format_check` checks the entire repo whereas the script only checks the diff? I think the issue is that we started linting after having some significant amount of code written so many files do not respect the style guide.